Ahorn/entities/lobbyMapWarp.jl

```diff
@@ -0,0 +1,43 @@
+module CollabUtils2LobbyMapWarp
+
+using ..Ahorn, Maple
+
+const default_sprite = "decals/1-forsakencity/bench_concrete"
+const default_activate_sprite = "CollabUtils2/characters/sitBench"
+
+@mapdef Entity "CollabUtils2/LobbyMapWarp" LobbyMapWarp(
+    x::Integer, y::Integer,
+    warpId::String="", icon::String="", dialogKey::String="",
```

Comment:
why is the default icon not given here?

Reply:
Because I missed it :pausefrogeline:

Reply:
As stated in the tooltip:

> The icon to display for this warp. Leaving this empty will default to the icon defined in the controller.

Reply:
If it has a default entered then the property should be removed from the controller since it will no longer do anything. It seems really bad for mappers to have to change the icon for every warp individually.